### DISCUSSION TITLE
Suggestion: Add HTML support to Change Log entries

### DISCUSSION THREAD

**Discussion by VWACBrick** [Azure/api-management-developer-portal#2545] [Feature request]
### Suggestion: 
When entering the text for a Change Log, it would be handy if it could render HTML in the final page. 

**Use Case:**  If there is a need to link out to external blog post, KB, or some other page that contains more details of why this change was needed.  

![image](https://user-images.githubusercontent.com/65634611/83454849-b36ebe00-a411-11ea-828d-a37ff84bb516.png)

**Note**: The API Settings/Description field will allow HTML and render it as HTML in the page. 

![image](https://user-images.githubusercontent.com/65634611/83454486-1d3a9800-a411-11ea-8c12-76a94bf75719.png)
![image](https://user-images.githubusercontent.com/65634611/83454568-393e3980-a411-11ea-8bdc-508f73967fc9.png)

**Comment by azaslonov** [comment]
Totally makes sense. Thank you.

**Comment by ilspiert** [comment]
Or instead of HTML use Markdown. This feature can ba mapped as well to the API Changelog page, where it would support writing better changelogs, or reverence to an additional documentation.
